Patch from Alex Williamson in
http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=350185 support of vmm= in the
elilo script

--- elilo/debian/elilo.sh.novmm	2006-03-16 07:49:19.000000000 -0500
+++ elilo/debian/elilo.sh	2006-03-16 07:48:47.000000000 -0500
@@ -364,6 +364,8 @@
 		sed 's/^image[[:space:]]*=[[:space:]]*//' | grep -v ':'`
     initrdfiles=`grep '^[[:space:]]*initrd[[:space:]]*=' $bootconf | \
 		sed 's/.*=[[:space:]]*//' | grep -v ':'`
+    vmmfiles=`grep '^[[:space:]]*vmm[[:space:]]*=' $bootconf | \
+		sed 's/.*=[[:space:]]*//' | grep -v ':'`
 
     ## Point of no return, removing the old EFI/debian tree
     rm -rf $TARGET/$EFIROOT
@@ -398,6 +400,7 @@
 	-e "s|\t| |g" \
 	-e "s|\(^image[[:space:]]*=[[:space:]]*\)\([^:]*\)$|\1$EFIROOT\2|" \
 	-e "s|\(^[[:space:]]*initrd[[:space:]]*=[[:space:]]*\)\([^:]*\)$|\1$EFIROOT\2|" \
+	-e "s|\(^[[:space:]]*vmm[[:space:]]*=[[:space:]]*\)\([^:]*\)$|\1$EFIROOT\2|" \
 	< "$bootconf" > "$TARGET/$EFIROOT/$CFFILE"
     if [ $? != 0 ] ; then
 	echo 1>&2 "$PRG: An error occured while writing to $boot"
@@ -406,7 +409,7 @@
 
     [ "$DEBUG" = 1 ] && echo "----" && cat "$TARGET/$EFIROOT/$CFFILE" && echo "----"
 
-    for i in $imagefiles $initrdfiles; do
+    for i in $imagefiles $initrdfiles $vmmfiles; do
 	[ "$VERBOSE" = 1 ] && echo "$PRG: Installing $i on $boot..."
 	if [ -f $i ]; then
 	    mkdir -p `dirname "$TARGET/$EFIROOT/$i"`
@@ -575,7 +578,9 @@
 		sed 's/^image[[:space:]]*=[[:space:]]*//' | grep -v ':'`
     initrdfiles=`grep '^[[:space:]]*initrd[[:space:]]*=' $bootconf | \
 		sed 's/.*=[[:space:]]*//' | grep -v ':'`
-    bytesneeded=`cat $imagefiles $initrdfiles $install $bootconf 2>/dev/null | wc -c`
+    vmmfiles=`grep '^[[:space:]]*vmm[[:space:]]*=' $bootconf | \
+		sed 's/.*=[[:space:]]*//' | grep -v ':'`
+    bytesneeded=`cat $imagefiles $initrdfiles $vmmfiles 2>/dev/null | wc -c`
     # convert to KB, allowing 5% overhead
     kbneeded=$(( bytesneeded / 1024 + bytesneeded / 20480 ))
     kbavailable=$(df -P -k $TARGET | sed -n "s|^$boot[[:space:]]\+[0-9]\+[[:space:]]\+[0-9]\+[[:space:]]\+\([0-9]\+\).*$|\1|p")
